<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame{
border-radius:5px;border-image: url(:/image/background/bakeground.jpg)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>211</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgba(255, 255, 255,150);
border-style:outset;

border-width:0px;

border-radius:10px;
border-color: rgba(0,0,0,100);
font:bold 14px;
color:rgb(180,180,180);
padding:6px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-image: url(:/image/btnimg/sousuo.png)46 repeat;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;


}

QPushButton:hover{

	background-image: url(:/image/btnimg/rsousuo.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/btnimg/sousuo.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>560</y>
       <width>1071</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/image/image/圆形 (9).png);
  margin: -6 0px; 
    }
      
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="now_time_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>640</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>640</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>/ 00:00:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/btnimg/quit.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="min_btn_2">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/btnimg/max.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="max_btn_2">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/btnimg/min.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="Lrc_list">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>851</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="music_name_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>590</y>
       <width>241</width>
       <height>64</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="last_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton
{

background: transparent;
background-image: url(:/image/btnimg/Lastshangyiqu.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;
border:none;



}

QPushButton:hover{

background-image: url(:/image/btnimg/rLastshangyiqu.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/btnimg/dLastshangyiqu.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="play_Btn">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-image: url(:/image/btnimg/Nextxiayiqu.png)46 repeat;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;


}

QPushButton:hover{

	background-image: url(:/image/btnimg/rNextxiayiqu.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/btnimg/dNextxiayiqu.png)46 repeat;
background-repeat:no-repeat;
background-position:center center;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="Name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>600</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/btnimg/cod.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>600</y>
       <width>351</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>52</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-image: url(:/image/btnimg/Volumeyinliang.png)32 repeat;
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;



</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_world">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton
{

background: transparent;
background-image: url(:/image/btnimg/songworld.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;
border:none;



}

QPushButton:hover{

background-image: url(:/image/btnimg/rsongworld.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/btnimg/songworld.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton
{

background: transparent;
background-image: url(:/image/btnimg/ListCycleliebiaoxunhuan.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;
border:none;



}

QPushButton:hover{

background-image: url(:/image/btnimg/rListCycleliebiaoxunhuan.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image:url(:/image/btnimg/ListCycleliebiaoxunhuan.png)32 repeat;
background-repeat:no-repeat;
background-position:center center;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>520</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/background/111.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}




</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
